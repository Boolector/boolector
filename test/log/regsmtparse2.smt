(benchmark ddregparsesmt0
:logic QF_BV
:formula
(let (?a true) b))
