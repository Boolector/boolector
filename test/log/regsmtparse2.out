log/regsmtparse2.smt:5: 'b' undefined
